tap_pipedrive/streams/pipelines.py

```diff
@@ -5,4 +5,5 @@ class PipelinesStream(PipedriveStream):
     endpoint = 'pipelines'
     schema = 'pipelines'
     key_properties = ['id', ]
-    state_field = 'update_time'
```

Comment:
So tables like these had a state_field but were operating full_table anyway?

Reply:
@KAllan357 We talked about this on-call, here is the doc link of changes incorporated in Pipedrive streams: https://talend365-my.sharepoint.com/:w:/g/personal/kpanchal_talend_com/EV4SU6EM5khDmDobC-hIXc4Bmv6cx-mx7NCOVLeCkS1HXg?e=B4Pohm